test/execute_build_test.go

```diff
@@ -70,148 +71,149 @@ func testCreateImage(t *testing.T, when spec.G, it spec.S) {
 		for _, tag := range cfg.generatedImageNames {
 			deleteImageTag(t, tag)
 		}
+		deleteNamespace(t, clients, testNamespace)
```

Comment:
The only reason not to keep this is to ensure we can investigate failing acceptance tests. I hate when the tests fail because of the namespace problem, but I'd rather we dont delete the namespace in the after

Reply:
@matthewmcnew I agree. I'll implement a wait.